model/integration_action.go

```diff
@@ -200,18 +200,21 @@ type Dialog struct {
 }
 
 type DialogElement struct {
-	DisplayName string               `json:"display_name"`
-	Name        string               `json:"name"`
-	Type        string               `json:"type"`
-	SubType     string               `json:"subtype"`
-	Default     string               `json:"default"`
-	Placeholder string               `json:"placeholder"`
-	HelpText    string               `json:"help_text"`
-	Optional    bool                 `json:"optional"`
-	MinLength   int                  `json:"min_length"`
-	MaxLength   int                  `json:"max_length"`
-	DataSource  string               `json:"data_source"`
-	Options     []*PostActionOptions `json:"options"`
+	DisplayName      string               `json:"display_name"`
+	Name             string               `json:"name"`
+	Type             string               `json:"type"`
+	SubType          string               `json:"subtype"`
+	Default          string               `json:"default"`
+	Placeholder      string               `json:"placeholder"`
+	HelpText         string               `json:"help_text"`
+	Optional         bool                 `json:"optional"`
+	MinLength        int                  `json:"min_length"`
+	MaxLength        int                  `json:"max_length"`
+	DataSource       string               `json:"data_source"`
+	Options          []*PostActionOptions `json:"options"`
+	DependsOn        string               `json:"depends_on"`
+	DispatchOnChange bool                 `json:"dispatch_on_change"`
```

Comment:
@hanzei `DispatchOnChange` is used to notify the server when a given value in the form has been selected. 

For example in the Jira create issue modal: If a project is selected, tell the server about the selection so it can create more form elements with its response.